kimchi/src/circuits/polynomials/varbasemul.rs

```diff
@@ -171,22 +187,22 @@ fn single_bit_witness<F: FftField>(
     w: &mut [Vec<F>; COLUMNS],
     row: usize,
     b: Variable,
-    base: CurveVar,
+    base: &Point<Variable>,
     s1: Variable,
-    input: CurveVar,
-    output: CurveVar,
+    input: &Point<Variable>,
+    output: &Point<Variable>,
     b_value: F,
     base_value: (F, F),
```

Comment:
No, because those are `expr::Variable`, which constains only `col,row`.  Think of them as placeholders (expr stuff) and this function translates them into values using environment which will either be `F`s or `Expr<ConstantExpr<F>>`s depending on the context.